hikari/impl/rest.py

```diff
@@ -1481,28 +1481,26 @@ async def delete_messages(
             # (albeit at a cost of maybe a couple-dozen milliseconds per call),
             # I am just gonna invoke these sequentially instead.
             try:
-                if len(pending) == 1:
-                    message = pending[0]
+                if len(chunk) == 1:
+                    message = chunk[0]
                     try:
                         await self.delete_message(channel, message)
-                    except errors.NotFoundError as exc:
+                    except errors.NotFoundError as e:
```

Comment:
`e` gives no context to what it may refer to.

Please change it back to `exc` or `ex`, which is at least an acronym to "exception"